velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -144,7 +144,8 @@ class PlanBuilder {
       const std::unordered_map<std::string, std::string>& columnAliases = {},
       const std::vector<std::string>& subfieldFilters = {},
       const std::string& remainingFilter = "",
-      const RowTypePtr& dataColumns = nullptr);
```

Comment:
Will use the TableScanBuilder if that gets merged first
https://github.com/facebookincubator/velox/pull/7463